lesson07/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
+  </head>
+  <body>
+    <div id="js-loading">
+      <img src="loading-circle.gif" alt="ローディング画像">
+    </div>
```

Comment:
I put the load image in HTML.
Should the load image be generated by JavaScript?